### DISCUSSION TITLE
preview version for Zotero 7

### DISCUSSION THREAD

**Discussion by volatile-static** [volatile-static/Chartero#36] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/volatile-static/Chartero/releases/tag/2.0.0-dev'>preview version for Zotero 7</a>.</em>

**Comment by 11good11** [comment]


**Reply by volatile-static** [reply]
#44 #43 等更新，谢谢

**Comment by 1tonsilla** [comment]
很漂亮 ，爱了爱了！无意间看到的！
